cypress/integration/[T01] Cypress Sink/[T01-02]input.spec.js

```diff
@@ -0,0 +1,49 @@
+// Imports
+import MethodsPage from "../../support/Pages/allMethodsList.js";
+import TypePage from "../../support/Pages/typePage.js";
+
+import {
+  checkVisiblity,
+  checkAmountOfElements,
+  checkIfValue,
+} from "../../support/Utils/assertions.js";
+
+// Initialization
+const methodsPage = new MethodsPage();
+const typePage = new TypePage();
+
+context("Checking fixture method", () => {
```

Comment:
czemu _context_ zamiast _describe_?

Reply:
można używać zamiennie tak naprawdę - to tylko hooki, tak samo `it()` można zamiennie używać z `specify()`

Reply:
to poprawiłbym tutaj na describe żeby wszędzie było spójnie